packages/types/src/Metadata/latest.json

```diff
@@ -1452,12 +1490,34 @@
             "modifier": 0,
             "type": {
               "key": "ReferendumIndex",
-              "value": "(BlockNumber,Proposal,VoteThreshold)"
+              "value": "(ReferendumInfo)"
```

Comment:
This one is a weird one, should simply be ReferendumInfo?

Reply:
It comes from 

```
/// Information concerning any given referendum.
		pub ReferendumInfoOf get(referendum_info): map ReferendumIndex => Option<(ReferendumInfo<T::BlockNumber, T::Proposal>)>;
```

It think if you look at the original, it would make sense why the Tuple was there. But here Tuple and no-Tuple are equivalent. But yes, weird.

Reply:
Hmm, so actually, here it should be `Option<(ReferendumInfo)>`

Reply:
Since it is storage, it doesn't make much difference, they all come back as either `null` or the value. However, it may point to something funny in the parsing and/or option just stripped out (I think the original was like that - Option was actually just ignored in `Type.ts`).